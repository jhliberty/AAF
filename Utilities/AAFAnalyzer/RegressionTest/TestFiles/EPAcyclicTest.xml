<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE aaf-file SYSTEM "EPAcyclicTest.dtd">
<aaf-file>
 <clip name="MasterMob1">
  <static-mob-slot>
   <source-clip length="0" track-type="audio">
    <file-source name="File Source 1" sample-rate-denominator="24" sample-rate-numerator="600">
     <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
      <source-clip length="2" track-type="video">
       <file-source name="File Source 2" sample-rate-denominator="24" sample-rate-numerator="600" parent-src-id="MasterMob1">
        <timeline-mob-slot edit-rate-denominator="1" edit-rate-numerator="1">
         <eoc length="0" track-type="none"/>
        </timeline-mob-slot>
       </file-source>
      </source-clip>
     </timeline-mob-slot>
    </file-source>
   </source-clip>
  </static-mob-slot>
 </clip>
</aaf-file>
