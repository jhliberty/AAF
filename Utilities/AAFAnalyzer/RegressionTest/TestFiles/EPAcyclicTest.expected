        Name:   Acyclic Analysis Test
        Desc:   Verify that source references are not cyclic.
        Result: Failed
        Reason: 
        Covered Requirements: None
        Noted Requirements: None
        Passing Requirements: None
        Info Requirements: None
        Warning Requirements: None
        Failing Requirements: REQ_EP_256

            Name:   Acyclic Derivation Chain
            Desc:   A mob derivation chain shall not contain a cycle.
            Result: Failed
            Reason: A cycle was detected amoung the referenced objects in the file.
            Covered Requirements: None
            Noted Requirements: None
            Passing Requirements: None
            Info Requirements: None
            Warning Requirements: None
            Failing Requirements: REQ_EP_256
            Detail: Nodes involved in the cycle:
            Detail: Node: 1  Object - Header
            Detail: Node: 2  Object - ContentStorage
            Detail: Node: 3  Object - MasterMob
            Detail: 	Name - MasterMob1
            Detail: 	MobID - 01010101010101010101010101010101-00000001-0001-0001-0101010101010101
            Detail: Node: 4  Object - StaticMobSlot
            Detail: 	SlotID - 1
            Detail: Node: 5  Object - SourceClip
            Detail: 	SourceID - 02020202020202020202020202020202-00000002-0002-0002-0202020202020202
            Detail: Node: 6  Object - SourceMob
            Detail: 	Name - File Source 1
            Detail: 	MobID - 02020202020202020202020202020202-00000002-0002-0002-0202020202020202
            Detail: Node: 7  Object - TimelineMobSlot
            Detail: 	SlotID - 1
            Detail: Node: 8  Object - SourceClip
            Detail: 	SourceID - 01010101010101010101010101010101-00000001-0001-0001-0101010101010101
            Detail: Node: 3  Object - MasterMob
            Detail: 	Name - MasterMob1
            Detail: 	MobID - 01010101010101010101010101010101-00000001-0001-0001-0101010101010101

